### PR TITLE
Set union

### PR DESCRIPTION
**Pull request by mistakeets**
finished set union algorithm

### CODE REVIEW

File: src/setUnion.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export default function setUnion(arr1, arr2) {
+  const arr = arr1.concat(arr2).sort()
+  for (var i=0; i<arr.length; i++){
+    if (arr[i] === arr[i+1]){
+      arr.splice(arr[i], 1)
+    }
+  } return arr
+}
```
Comment (devonwesley) [inline src/setUnion.js:8]: Sweet nice and concise goodjob!

File: test/setUnion_test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    expect(setUnion([1,2,3,4], [5,6,7,8])).to.deep.equal([1,2,3,4,5,6,7,8])
+  })
+
+  it('array length is equal to non-duplicate numbers', function(){
+    let arrayLength = setUnion([1,2,3,4], [2,4,6,8]).length
+    console.log('length', arrayLength);
+    expect(arrayLength).to.equal(6)
+  })
+
+})
```
Comment (devonwesley) [inline test/setUnion_test.js:24]: Test look good.

**Comment by devonwesley** [comment]
Go ahead and merge this look good.

File: src/setUnion.js (hunk 1 above)
Comment (jaredatron) [reply src/setUnion.js:8]: I would recommend a different formatting

```js
export default function setUnion(arr1, arr2) {
  const arr = arr1.concat(arr2).sort()
  for (var i = 0; i < arr.length; i++){
    if (arr[i] === arr[i+1]){
      arr.splice(arr[i], 1)
    }
  } 
  return arr
}
```

Also are you sure you can `splice` the array you're looping over? Is that intentional to your logic here?

File: test/setUnion_test.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  })
+
+  it('if no duplicates it concats and sorts arrays', function(){
+    expect(setUnion([1,2,3,4], [5,6,7,8])).to.deep.equal([1,2,3,4,5,6,7,8])
+  })
+
+  it('array length is equal to non-duplicate numbers', function(){
+    let arrayLength = setUnion([1,2,3,4], [2,4,6,8]).length
+    console.log('length', arrayLength);
+    expect(arrayLength).to.equal(6)
```
Comment (jaredatron) [inline test/setUnion_test.js:21]:
```js
let union = setUnion([1,2,3,4], [2,4,6,8])
expect(union).to.have.length(6)
```

**Review by jaredatron — COMMENTED** [review]
otherwise looks good

File: test/setUnion_test.js (hunk 3 above)
Comment (jaredatron) [reply test/setUnion_test.js:21]: CC: @Jusdev89

**Pull request merged**